modules/inflowwind/src/InflowWind.f90

```diff
@@ -290,8 +284,17 @@ SUBROUTINE InflowWind_Init( InitInp, InputGuess, p, ContStates, DiscStates, Cons
    ! Set flow field input data based on wind type
    !----------------------------------------------------------------------------
 
+   ! If flowfield is allocated, deallocate and allocate again to clear old data
+   if (associated(p%FlowField)) deallocate(p%FlowField)
```

Comment:
`allocated` or `associated` ?

Reply:
Since `p%FlowField` is now a pointer, `associated` is used to check if the pointer has memory associated with it. `allocated` is used for allocatable arrays.

Reply:
That makes sense.  At first glance it looked rather strange.